File: python/sequential.py
import asyncio

async def sample(delay, text):
    i = 0
    while(i<5):
        await asyncio.sleep(delay)
        print(text)
        i+=1

async def main():
    await sample(2, "Hello")
    await sample(2, "World")

asyncio.run(main())

File: js/sequential.js
async function sleep(delay) {
  return new Promise(resolve => {
    setTimeout(() => resolve(), delay * 1000);
  });
}

async function delayedPrint(delay, text) {
  let i = 0;
  while (i < 5) {
    await sleep(delay);
    console.log(text);
    i++;
  }

  return Promise.resolve();
}

async function main() {
  await delayedPrint(2, "First");
  await delayedPrint(2, "Second");
}

main();
